src/devices/vdml_imu.cpp

```diff
@@ -65,7 +65,7 @@ pros::ImuStatus Imu::get_status() const {
 }
 
 bool Imu::is_calibrating() const {
-	return (int)get_status() & (int)(pros::ImuStatus::calibrating);
+	return (int)get_status() == (int)(pros::ImuStatus::calibrating);
```

Comment:
This introduces a bug where the condition will evaluate to false if the IMU is in any position other than upright, as the orientation bits from `get_status` will no longer exactly match the value of the `calibrating` enum. See https://github.com/purduesigbots/pros/pull/643